src/main/java/frc/robot/Constants.java

```diff
@@ -27,7 +27,8 @@ public AimModes previous() {
     },
     EJECT(2500.0, 0.0),
     LAUNCH(0.0, 0.0),
-    TARMAC(2530.0, 0.0),
+    TARMAC(2480.0, 0.0),
+    ATARMAC(2480.0, 0.0),
```

Comment:
Same comment.  These are the same, wonder if they should be different.  Is one adjusted in case of over-inflated balls?

Reply:
Answered in other PR